src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -216,5 +180,20 @@ private string GetProjectName()
 
             return projectName;
         }
+
+        public IEnumerable<ITagSpan<IErrorTag>> GetTags(NormalizedSnapshotSpanCollection spans)
+        {
+            return Enumerable.Empty<ITagSpan<IErrorTag>>();
+        }
+
+        public event EventHandler<SnapshotSpanEventArgs> TagsChanged;
+
+        public void Dispose()
+        {
+            document.FileActionOccurred -= SafeOnFileActionOccurred;
+            textBuffer.Properties.RemoveProperty(typeof(IIssueTracker));
```

Comment:
It used to be `textBuffer.Properties.RemoveProperty(typeof(TextBufferIssueTracker));` which I believe was a mistake, since that's not how we add the property.